util/pkg/architectures/architectures.go

```diff
@@ -29,6 +29,12 @@ const (
 	ArchitectureArm64 Architecture = "arm64"
 )
 
+type GPUVendor string
+
+const (
+	GPUVendorNvidia GPUVendor = "nvidia"
+)
+
```

Comment:
This has absolutely nothing to do with architecture. It is a sidenote at best that should be confined to a few files in nodeup.